### PR TITLE
feature: created a react app boilerplate

### PR DESCRIPTION
**Pull request by mvaldes42**
Created a react app boiler plate via npx create-react-app to test out the pull request system.
Tuto here : https://www.digitalocean.com/community/tutorials/how-to-deploy-a-react-application-with-nginx-on-ubuntu-20-04

### CODE REVIEW

**Review by fcavillo — COMMENTED** [review]
Very nice work, I'm trying to leave a comment without approving ! 
I wish I could see the favicon but it won't load. 
And why does the rel attribute in the index.html say apple touch, it's dead

**Review by mvaldes42 — COMMENTED** [review]
Lets try this out !

File: front/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
```
[the remark this answers is not in the document]
Comment (mvaldes42) [reply front/README.md:16]: Great !

**Review by fcavillo — APPROVED** [review]
It works great, congrats mvaldes

**Pull request merged**